### PR TITLE
docs: fixed numbering of license bullets and corrected indent

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
This PR fixes the license list numbering behaving incorrectly 

image

### CODE REVIEW

File: website/content/docs/nia/enterprise/license.mdx [hunk 1]
```diff
@@ -27,13 +27,13 @@ Choose one of the following methods (in order of precedence) to set the license:
 export CONSUL_LICENSE=<LICENSE_VALUE>
 ```
 
-2. Set the `CONSUL_LICENSE_PATH` environment variable to the path of the file containing the license.
+1. Set the `CONSUL_LICENSE_PATH` environment variable to the path of the file containing the license.
 
 ```shell-session
 export CONSUL_LICENSE_PATH=<PATH>/<TO>/<FILE>
 ```
```
Comment (blake) [inline website/content/docs/nia/enterprise/license.mdx:35]: I believe the numbering issues would be resolved if the code blocks are indented.
```suggestion
1. Set the `CONSUL_LICENSE_PATH` environment variable to the path of the file containing the license.

    ```shell-session
    export CONSUL_LICENSE_PATH=<PATH>/<TO>/<FILE>
    ```

```

Comment (karl-cardenas-coding) [reply website/content/docs/nia/enterprise/license.mdx:35]: @blake I just tried that and the extra indentation has no effect as it's already indented from the bullet anchor (see image). The bullets should also be all `1.` per markdown syntax. 

image

**Comment by lornasong** [comment]
Thanks for the PR! When I visit the preview link, I still see all 1s numbering and no indent (I'm not sure if you decided to go with/without indents). I'm looking [here](https://consul-2vysj561x-hashicorp.vercel.app/docs/nia/enterprise/license#setting-the-license). Please let me know if I'm looking in the wrong place.

**Comment by karl-cardenas-coding** [comment]
> Thanks for the PR! When I visit the preview link, I still see all 1s numbering and no indent (I'm not sure if you decided to go with/without indents). I'm looking [here](https://consul-2vysj561x-hashicorp.vercel.app/docs/nia/enterprise/license#setting-the-license). Please let me know if I'm looking in the wrong place.

That's really bizarre because locally (local dev server) it renders correctly.
But yes, you are looking at the right [link](https://consul-2vysj561x-hashicorp.vercel.app/docs/nia/enterprise/license)

**Comment by lornasong** [comment]
Gotcha. Interesting that it renders differently. I saw your screenshot in the description so guessed that you must have got it working in some way, shape, or form

**Comment by karl-cardenas-coding** [comment]
@lornasong I posted a question in the marketing dev channel 
https://hashicorp.slack.com/archives/C5FSPUGDS/p1630605639031500

**Comment by karl-cardenas-coding** [comment]
@blake Interesting finding, if I indent as you suggested it renders correctly. 
@lornasong go ahead and checkout the new [preview link](https://consul-6eadwcmyk-hashicorp.vercel.app/docs/nia/enterprise/license).

**Review by lornasong — APPROVED** [review]
Preview's numbering and indentations look nice 🎉

**Review by blake — APPROVED** [review]

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/440070.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit a2e6a15e936b4907bf6a1790d91da96ee8949997 onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit a2e6a15e936b4907bf6a1790d91da96ee8949997 onto `release/1.10.x` succeeded!

**Pull request merged**